src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -89,6 +90,19 @@ public bool Active
             }
         }
 
+        public virtual bool IsGridVisible
```

Comment:
Per our documentation standards, all new `public` members need some documentation. I think this is pretty straightforward:  
> A flag indicating whether the grid is visible in 3D.